sdk/core/perf/src/base_test.cpp

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "azure/perf/base_test.hpp"
+#include "azure/core/http/policies/policy.hpp"
+#include "azure/core/internal/http/pipeline.hpp"
+
+#include <functional>
+#include <string>
+#include <vector>
+
+using namespace Azure::Core::Http;
+using namespace Azure::Core::Http::Policies;
+using namespace Azure::Core;
+namespace {
```

Comment:
unnamed namespace?

Reply:
yes, we use the unamaned namespace from within .cpp impl files